frontend/src/components/PLAYGROUND/ReportSelector/handleExport.tsx

```diff
@@ -0,0 +1,49 @@
+import { ConvertToStringItem, UpdatedSearchParams } from "types";
+import { apiAxios } from "utils";
+
+const convertValueToString = (item: ConvertToStringItem) => {
```

Comment:
could delete lines 5-7 if you write line 4 as:

`const convertValueToString = (item = "") => {`